### PR TITLE
Run page improvements/fixes

### PR DESCRIPTION
**Pull request by znicholasbrown**
PR Checklist:

- [x] add a short description of what's changed to the top of the `CHANGELOG.md`
- [ ] add/update tests (or don't, for reasons explained below)

## Describe this PR
Fixes issues with pre-populating and resetting job templates and task definitions for run configs, and resets option args when navigating between options, from a stored initial value.

### CODE REVIEW

File: src/components/CustomInputs/MultiLineInput.vue [hunk 1]
```diff
@@ -41,17 +43,46 @@ export default {
...
 
         // Use next tick to make sure the json input element exists
         this.$nextTick(() => {
           this.$refs['json-input'].validateJson()
         })
+      } else {
+        if (this.jsonInput === '{}') {
+          this.yamlInput = ''
+        } else {
+          this.yamlInput = stringify(JSON.parse(this.jsonInput))
```
Comment (zhen0) [inline src/components/CustomInputs/MultiLineInput.vue:63]: Not a major issue but do we need a try/catch here?  If I add yaml to the json input by mistake and try to switch to yaml, the input vanishes. 
Screen Recording 2021-05-18 at 4.53.10 PM

**Review by zhen0 — COMMENTED** [review]
One small question/suggestion @znicholasbrown

File: src/components/CustomInputs/MultiLineInput.vue (hunk 1 above)
Comment (zhen0) [reply src/components/CustomInputs/MultiLineInput.vue:63]: https://user-images.githubusercontent.com/40272060/118722313-03e55400-b7fa-11eb-876a-ebd9dd782167.mov

Comment (znicholasbrown) [reply src/components/CustomInputs/MultiLineInput.vue:63]: Hm yeah I think a try catch makes sense here - will add that

Comment (znicholasbrown) [reply src/components/CustomInputs/MultiLineInput.vue]: Updated 👍🏻

**Review by zhen0 — APPROVED** [review]
Thanks for the changes @znicholasbrown

**Pull request merged**